addon/components/frost-list.js

```diff
@@ -129,17 +129,6 @@ export default Component.extend({
 
   // == Lifecycle Hooks =======================================================
 
-  didUpdateAttrs ({newAttrs}) {
```

Comment:
We don't need the frost scroll with the `vertical-collection`. The vertical collection has the scrolling capability.